tests/test_telegramobject.py

```diff
@@ -138,6 +138,26 @@ def test_to_dict_api_kwargs(self):
         to = TelegramObject(api_kwargs={"foo": "bar"})
         assert to.to_dict() == {"foo": "bar"}
 
+    def test_to_dict_recursion(self):
+        class Recusive(TelegramObject):
+            def __init__(self):
+                super().__init__()
+                self.recursive = "recursive"
+
+        class SubClass(TelegramObject):
```

Comment:
maybe we can add `__slots__` to one of these two and a comment "doesn't have `__slots__`, so has `__dict__`" to the other? just to emphasize that both should work :)